docs/binary-tarball-names.md

```diff
@@ -1,5 +1,35 @@
 # Binary tarball file names available
 
+For recent versions of Percona XtraBackup, the tar files are organized by the `glibc2` version. You can find this version on your operating system with the following command:
+
+```{.bash data-prompt="$"}
+$ ldd --version
+```
+
+??? example "Expected output"
+    ```text
+    ldd (Ubuntu GLIBC 2.35-0ubuntu3.1) 2.35
+    Copyright (C) 2022 Free Software Foundation, Inc.
+    This is free software; see the source for copying conditions.  There is NO
+    warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    Written by Roland McGrath and Ulrich Drepper.
+    ```
+
+If the `glibc2` version from your operating system is not listed, then this Percona Server for MySQL version does not support that operating system.
+
+## Binary tarball file name organization
+
+The following lists the platform and the associated full binary file name used by Percona Server for MySQL tar files {{release}}.
+
+| Platform             | Percona XtraBackup tarball name                   | glibc2 version |
+|----------------------|--------------------------------------------------------------|-----------|
+| Ubuntu 24.04         | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.39.tar.gz     | glibc2.39 |
+| Ubuntu 22.04         | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.35.tar.gz     | glibc2.35 |
+| Ubuntu 20.04         | Percona-XtraBackup-{{release}}-Linux.x86_64.glibc2.31.tar.gz     | glibc2.31 |
```

Comment:
24.04/ Debian 13 (trixie)
22.04/Debian 12 (bookworm)
20.04/Debian 11 (bullseye)

Reply:
We don't use the nicknames

Reply:
Also Debian 13 is not released yet and is not expected until 2025.